src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,90 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+    private static final int RESIZE_UP_MULTIPLICATOR = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int sizeThreshold;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
Using a raw type when creating a new array of Nodes is not type-safe. Use the diamond operator to ensure type correctness. It should be 'new Node<>[DEFAULT_INITIAL_CAPACITY]' to avoid unchecked assignment warnings.